constants/users.js

```diff
@@ -19,6 +19,8 @@ const ALLOWED_FILTER_PARAMS = {
 
 const DOCUMENT_WRITE_SIZE = 500;
 
+const MAX_IN_OPERANDS = 30;
```

Comment:
- What does `MAX_IN_OPERANDS` mean?

Reply:
It's the maximum number of values allowed in firestore `IN` clause for querying data. Changed it to `FS_IN_CLAUSE_SIZE`. Let me know if it makes more sense now.

Reply:
- Can we removthe ambiguity around `FS` and name it  `FIRESTORE_IN_CLAUSE_SIZE` ?

Reply:
done.